### PR TITLE
Toolbar Reset on Mobile and Desktop 

### PR DESCRIPTION
**Pull request by thivya-deriv**
Card: https://app.clickup.com/t/20696747/BOT-1950
Desc: This is to cover test on reseting workspace from toolbar

### CODE REVIEW

File: cypress/e2e/smoke/dbot/Toolbar.cy.js [hunk 1]
```diff
@@ -23,21 +23,25 @@ describe('QATEST-99418: Verify toolbar on bot builder page', () => {
       cy.c_switchToDemoBot()
       botBuilder.openBotBuilderTab()
       cy.c_skipTour()
+      //Save strategy from toolbar
```
Comment (rajendra-deriv) [inline cypress/e2e/smoke/dbot/Toolbar.cy.js:26]: You can remove the commented line as there is a Logger message which states the same.

File: cypress/e2e/smoke/dbot/Toolbar.cy.js [hunk 2]
```diff
@@ -23,21 +23,25 @@ describe('QATEST-99418: Verify toolbar on bot builder page', () => {
...
       cy.log('Saving strategy from Toolbar')
       botBuilder.changeMarketOnBlocklyWorkspace(1, 'Stock Indices')
       botBuilder.changeMarketOnBlocklyWorkspace(2, 'European indices')
       botBuilder.saveStrategyFromToolbar(strategyName)
       botDashboard.goToDashboard()
       botDashboard.strategySaveStatus(strategyName).should('have.text', 'Local')
+      //Import strategy from toolbar
```
Comment (rajendra-deriv) [inline cypress/e2e/smoke/dbot/Toolbar.cy.js:33]: You can remove the commented line as there is a Logger message which states the same.

**Review by rajendra-deriv — APPROVED** [review]

**Review by abuzar-deriv — CHANGES_REQUESTED** [review]
Looks good but please add testid QATEST-4238 to the file.

**Review by abuzar-deriv — APPROVED** [review]
LGTM

**Review by abuzar-deriv — APPROVED** [review]
LGTM

**Pull request merged**